app/editor/src/hooks/api-editor/constants/ScheduleType.ts

```diff
@@ -1,8 +1,8 @@
 export enum ScheduleType {
   /** Repeating will run a continuously repeating schedule */
-  Continuous = 0,
+  Continuous = 1,
```

Comment:
Reminder that everyone will need to `make db-refresh` so that this doesn't cause issues with existing data.